Api/Helper/Csv.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Werkspot\BingAdsApiBundle\Api\Helper;
+
+class Csv
+{
+    const FILE_HEADERS = 10;
```

Comment:
10? Why not 11? 

I guess this is very specific to a specific bing report, we should probably put it somewhere else and make it a specific name